### PR TITLE
Search bar was created

### PR DESCRIPTION
**Pull request by BilokonYaroslav**
https://BilokonYaroslav.github.io/layout_search-bar-airbnb/)
https://BilokonYaroslav.github.io/layout_search-bar-airbnb/report/html_report/

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,139 @@
+@font-face {
+  font-family: Avenir;
+  font-weight: normal;
```
Comment (witflash) [inline src/style.css:3]: `font-weight: 300` if you use numeric value below

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+  top: 24px;
+  left: 25px;
+}
+
+.search__area {
+  border: none;
+  padding: 0;
+  /*position: absolute;
+  top: 24px;
+  left: 61px;*/
```
Comment (witflash) [inline src/style.css:49]: Remove comments from your code

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+  color: #3d4e61;
+  font-weight: normal;
+  text-shadow: none;
+  font-family: "Avenir", Times, serif;
+}
+
+.search__area:focus::placeholder {
+  font-weight: normal;
+  text-shadow: none;
+  color: #3d4e61;
```
Comment (witflash) [inline src/style.css:75]: You can move value to the CSS variable (--color-main, for instance) if you use it more than one time

File: src/style.css [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+
+.search-small__area {
+  position: absolute;
+  top: 9px;
+  left: 33px;
+  border: none;
+  color: #3d4e61;
+  font-weight: bold;
+  line-height: 19px;
+  /* font-weight: 400; */
```
Comment (witflash) [inline src/style.css:108]: Remove comment

File: src/style.css [hunk 5]
```diff
@@ -0,0 +1,139 @@
...
+  font-family: "Avenir", Times, serif;
+  font-size: 14px;
+}
+
+/*.search-small__area:hover {
+
+}*/
```
Comment (witflash) [inline src/style.css:115]: Leftover

**Review by witflash — CHANGES_REQUESTED** [review]
Good job.
Pay attention to my comments, please

**Comment by BilokonYaroslav** [comment]
Я не розумію чому посилання не працюють

**Comment by BilokonYaroslav** [comment]
Links are working!

**Review by Virass — APPROVED** [review]

**Pull request closed without merging**